src/components/molecules/ChatsList/ChatsList.scss

```diff
@@ -96,7 +96,7 @@ $primary: #005ee5;
 
 .private-messages-empty {
   width: 100%;
-  height: calc(100% - 114px);
+  height: calc(100% - 180px);
```

Comment:
This should calculate 100% - input height + footer height. TBD

Reply:
OK I'm a little confused by the TBD comment. Are you saying here that you don't want to merge this PR, or that it can be changed in a future PR?

Reply:
Another instance of `height: calc(100% - 180px)`